src/extensibility/node/package-validator.js

```diff
@@ -173,6 +173,15 @@ function findCommonPrefix(extractDir, callback) {
 }
 
 /**
+ * Strip the BOM from the file if it has one.
+ * @param {String} data The raw file contents.
+ * @return {String} The same data, sans the BOM.
+ */
+function stripBOM(data) {
+    return data.substr(data.indexOf("{")).replace(/\u0000/g, "");
```

Comment:
this looks a bit weird, I believe this implementation is better:
https://github.com/sindresorhus/strip-bom/blob/master/index.js